lib/src/commands/create.dart

```diff
@@ -126,6 +135,32 @@ class CreateCommand extends Command<int> {
     return projectName;
   }
 
+  /// Gets the organization name.
+  List<Map<String, String>> get _orgName {
+    final orgName =
+        (_argResults['org-name'] ?? 'com.example.verygoodcore') as String;
+    _validateOrgName(orgName);
```

Comment:
don't we only need to validate if it's not the default? I think we can return the default early if the user didn't provide a custom `org-name`.